usecase/person.go

```diff
@@ -0,0 +1,84 @@
+package usecase
+
+import (
+	"errors"
+	"github.com/labstack/echo/v4"
+	"github.com/pankecho/golang-bootcamp-2020/entity"
+)
+
+type Headers int
```

Comment:
By convention, all the descriptions should start with the name of the struct, interface, function, etc name